assets/admin_themes/Sea_Green/topbar/survey_actionbar.scss

```
@@ -147,3 +147,8 @@
     display: block !important;
   }
 }
+
+i.view-switch:focus {
+    outline        : 2px solid currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)